Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -113,6 +113,7 @@ LogManager::LogManager()
   m_log[LogType::DSP_MAIL] = {"DSPMails", "DSP Mails"};
   m_log[LogType::DSPINTERFACE] = {"DSP", "DSP Interface"};
   m_log[LogType::DVDINTERFACE] = {"DVD", "DVD Interface"};
+  m_log[LogType::DVDINTERFACE_AMMB] = { "DVD", "AMMB Interface" };
```

Comment:
The first value in the list is used as the key in the `Logger.ini` configuration file, and needs to be distinct. Presumably this will result in the AMMB setting overwriting the DVD Interface setting when dolphin is restarted.

```suggestion
  m_log[LogType::DVDINTERFACE_AMMB] = { "DVD_AMMB", "AMMB Interface" };
```